ONSdigital/eq-author-app
# Add validation count to logic tabs

### What is the context of this PR?

Adds the red dot to logic tab when there are errors due to routing or skip conditions

### How to review 

- Create a questionnaire
- Don't fill in the answer label
- Notice a red dot on the design tab
- Add an empty routing rule
- Notice an red dot on the logic tab
- Add an empty skip condition
- Notice the red dot hasn't changed
- Correct all the validation errors
- All red dots should be gone
- Also the numbered errors in the section nav correspond with the amount of errors present
